docs/src/services/relay.md

```diff
@@ -0,0 +1,23 @@
+# Control Plane Relay
+
+| services | ports | Protocol |
+|----------|-------|-----------|
+| ADS | 7800 | gRPC(IPv4) |
+| CPDS | 7900 | gRPC(IPv4) |
+
+For multi-cluster integration, Quilkin provides a `relay` service, that can be
+used with a multiple [control plane](./xds.md) services in different clusters to
+provide a unified "Aggregated Discovery Service" endpoint for [proxy](./proxy.md)
+services.
+
+To connect to a control plane to a relay, add the `--relay` flag to your control
+plane with the address of the relay. Then to connect a proxy service to the
+relay's ADS endpoint, you use the same `--management-server` argument for
+connecting to control planes.
+
```

Comment:
I'd love to see a simple example here for clarify - something someone could run locally on their machine.

Reading the docs I'm mentally trying to picture which if it's `quilkin proxy` has the `--relay` or if it's the `quilkin relay` or something else? (I'm pretty sure I get it, but I had to squint on it). Also to show which port should be connected to where. It looks like it's easy to mess up.

I was thinking something like (please make any corrections as needed):

---

As a simple local example (we'll disable `--no-admin` to avoid port conflicts), you can have:

To run the relay:
```shell
$ quilkin relay --no-admin
```

Connect a file control plane to the relay

```shell
$ quilkin manager file ./examples/proxy.yaml --no-admin --relay http://localhost:7900
```

No we can connect the proxy to the relay:

```shell
$ quilkin proxy --management-server http://localhost:7800
```